assets/js/geocodeAPI.js

```diff
@@ -0,0 +1,27 @@
+let apiBase = 'https://nominatim.openstreetmap.org/search?';
+
+function cityToLatLon(cityName) {
+    let requestURL = apiBase + `city=${cityName}&format=json`;
+
+    return fetch(requestURL)
+        .then(function (response) {
+            return response.json();
+        })
+        .then(function (data) {
+            console.log(data[0].display_name);
+            let geocodedArray = {
+                city: data[0].display_name.split(',')[0],
+                state: data[0].display_name.split(',')[2],
+                country: data[0].display_name.split(',')[3],
```

Comment:
the indicies should be `0`, `1`, and `2`.

Reply:
also, you can probably create the array and with split and then grab each part individually:
```js
let splitArray = data[0].display_name.split(',');
let geocodedArray = {
    city: splitArray[0],
    state: splitArray[1],
    country: splitArray[2],
    // ....
}```